database/postgres/data_generation/overviews_write.js

```diff
@@ -0,0 +1,28 @@
+const overviewsScript = require('./overviews_script');
+const fs = require('fs');
+
+const RECORDS_COUNT = 1000;
```

Comment:
A little nitpicky, but try to stick to a naming convention for all of your variables. All caps here